Added shaper function to the lib

trickstival opened the pull request:
By creating a simple wrapper around bind() it's possible to provide a clearer API.

This way there's no need for the user to pass null as the first parameter of bind nor understand why it's needed

posva commented:
I like this! The doc isn't right though, it would require to `import { shaper } from `. It also needs the typings, and a test would be nice. As with the other PR, tell me if don't want or cannot add typings, it's absolutely fine!

trickstival commented:
I've added a new test for the shaper func and fixed the `README.md` file, but I'm not sure how the typings work. I would have to do some research to learn how to use the `d.ts` files

posva commented:
Sorry, misclick. Don't worry about the typings, I can add them myself 🙂

posva commented on README.md:
```diff
@@ -46,8 +46,9 @@ const reshapedUser = shapify(
+import { shapify, shaper } from 'shapify'
 // create a function with the first parameter fixed
-const userShaper = shapify.bind(null, { text: 'fullname', value: 'id' })
+const userShaper = shaper({ text: 'fullname', value: 'id' })
```
```suggestion
const userShaper = shaper({ text: 'fullname', value: 'id' })
// equivalent to
const userShaper = shapify.bind(null, { text: 'fullname', value: 'id' })
```

posva commented:
I couldn't make the typings for this, I will open an issue and see if anybody can help
